docs/introduction.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Introduction
+id: introduction
+---
+
+# What is `granary`?
+
+Granary is a job runner for cloud-based geospatial machine learning.
+Its goal is to simplify running models and to track and serve
+the results of predictions. It puts a REST API between you and AWS Batch
+to simplify interactions that otherwise involve repeatedly checking AWS SDK
+documentation.
+
+## What's a `Model`?
+
+A model is a bundle of a human-readable name, some, AWS Batch configuration,
+and an argument validator.
+
+```scala mdoc
+import com.rasterfoundry.granary.datamodel._
+
+import io.circe.syntax._
+
+import java.util.UUID
+
+val model = Model.Create(
+  "A descriptive model name",
+  Validator(().asJson), // a Validator -- more on this below
+  "perfectAccuracyModel:1", // an AWS Batch job definition
+  "veryExpensiveOnDemandQueue" // an AWS Batch job queue
+)
+// What you'll POST to the API to create a model
+model.asJson.spaces2
```

Comment:
I was a little confused by this at first. If I am understanding this correctly it look like the "compiler" for the documentation actually runs the Scala code and appends the output.